### PR TITLE
CAA records fix

### PR DESCRIPTION
**Pull request by ArifPMan**
Abhijit,

Can you please review and approve this?

### CODE REVIEW

**Comment by ArifPMan** [comment]
Akshay, Can you please review and approve this request?

File: v6/postman/api_documentation/adding_and_verifying_custom_domains.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ Click **Proceed** to verify the domain.
 
 To verify a domain, check the box "I've added the TXT and CNAME records". Next click **Verify Domain**. A confirmation message appears stating domain has been verified. 
 
+**Note:** Postman uses LetsEncrypt as an SSL certificate provider to enable it to host public documentation on your domain. Postman generates a LetsEncrypt certificate successfully if your domain has no CAA records. If your domain has CAA records set, then you must explicitly allow LetsEncrypt to set certificates for your domain. To allow LetsEncrypt generate this certificate, refer to [LetsEncrypt Documentation](https://letsencrypt.org/docs/caa)
```
Comment (elssar) [inline v6/postman/api_documentation/adding_and_verifying_custom_domains.md]: >Postman generates a LetsEncrypt certificate successfully if your domain has no CAA records
This is incorrect. We do not generate a certificate, we order one from Lets Encrypt, which issues/generates it. As part of its verification process Lets Encrypt verifies the CAA records to ensure that it is allowed to issue certificates for that domain. If there is no CAA record then it is implicitly allowed to issue certificates, but if there is a CAA record, then it needs an explicit CAA record that allows Lets Encrypt to issue a certificate for that domain.

File: v6/postman/monitors/viewing_monitor_results.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ The [Monitors](https://monitor.getpostman.com/) page lists all your monitors (
...
-| **Average success rate** | The percent of successful runs out of the total runs in the timeframe. A run is successful only when all the tests pass during that run. You can also see a change in this value compared to the previous time period. |
-| **Average response time** | The average response time of all the requests over all the runs in milliseconds. You can also see the percentage change from the previous time period. |
+| **Status** | **Average success rate** | **Average response time** |
+| -----------| -------------------------| --------------------------|
+| The number of failed runs in the selected timeframe. A ‘Healthy’ status indicates there were no failures in any of the runs. | The percent of successful runs out of the total runs in the timeframe. A run is successful only when all the tests pass during that run. You can also see a change in this value compared to the previous time period. | The average response time of all the requests over all the runs in milliseconds. You can also see the percentage change from the previous time period. |
```
Comment (elssar) [inline v6/postman/monitors/viewing_monitor_results.md]: Why is this change in a PR to add information about CAA records in the documentation :thinking:

File: v6/postman/api_documentation/adding_and_verifying_custom_domains.md (hunk 1 above)
Comment (ArifPMan) [reply v6/postman/api_documentation/adding_and_verifying_custom_domains.md]: Updated per your f/b. Please review now.

File: v6/postman/monitors/viewing_monitor_results.md (hunk 2 above)
Comment (ArifPMan) [reply v6/postman/monitors/viewing_monitor_results.md]: Ignore this for now, I will take care of it.

**Pull request merged**